gravity-forms/gw-quantity-as-decimal.php

```diff
@@ -0,0 +1,128 @@
+<?php
+/**
+ * Gravity Wiz // Gravity Forms // Accept Decimal Values for Quantity Fields
+ *
+ * Allows you to accept decimal values in Quantity fields, converting any Quantity field into a weight field.
+ *
+ * @version   1.3
+ * @author    David Smith <david@gravitywiz.com>
+ * @license   GPL-2.0+
+ * @link      http://gravitywiz.com/enable-decimal-values-in-quantity-fields/
```

Comment:
Can you replace this with a plugin header?

Reply:
@spivurno  I have updated the snippet with a plugin header.

Reply:
Ah, my intent was only to replace the lines that started with `@` that have been highlighted above. Could you put the other stuff back?